apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -1,13 +1,52 @@
 import { findAllNewAppeals } from './validation.service.js';
 
+/**
+ * Given an item object transform it into - govuk/components/table/macro.njk" import govukTable format
+ *
+ * @param {*} item to do
```

Comment:
If you are not use JSDoc please don't add them. Is not a requirement for now.

Reply:
👍